api/wrangler.toml

```diff
@@ -5,4 +5,7 @@ compatibility_flags = ["nodejs_compat"]
 
 [vars]
 FRONTEND_URL = "https://webmonetization.org/tools/"
-AWS_PREFIX = "20250717"
\ No newline at end of file
+AWS_PREFIX = "20250717"

```

Comment:
Cleaning these vars will be done post-release, to not affect production versions.